server/trillian_log_server/main.go

```diff
@@ -28,100 +28,69 @@ import (
 	"github.com/google/trillian/crypto/keys"
 	"github.com/google/trillian/extension"
 	"github.com/google/trillian/monitoring"
-	"github.com/google/trillian/monitoring/metric"
 	"github.com/google/trillian/server"
-	"github.com/google/trillian/server/admin"
 	"github.com/google/trillian/storage/mysql"
 	"github.com/google/trillian/util"
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
 	"google.golang.org/grpc"
-	"google.golang.org/grpc/reflection"
 )
 
 var (
 	mySQLURI            = flag.String("mysql_uri", "test:zaphod@tcp(127.0.0.1:3306)/test", "Connection URI for MySQL database")
 	serverPortFlag      = flag.Int("port", 8090, "Port to serve log RPC requests on")
-	exportRPCMetrics    = flag.Bool("export_metrics", true, "If true starts HTTP server and exports stats")
-	httpPortFlag        = flag.Int("http_port", 8091, "Port to serve HTTP metrics on")
+	httpPortFlag        = flag.Int("http_port", 8091, "Port to serve HTTP metrics and REST requests on (negative means disabled)")
```

Comment:
Consider renaming this flag to something like `metrics_port`. 
Now that we have http and grpc endpoints, `http_port` is overloaded.